cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -209,7 +209,7 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodesList() {
     marker.header.frame_id = node_options_.map_frame;
     marker.color.b = 1.0;
     marker.color.a = 1.0;
```

Comment:
Okay. If I go with the procedural colours, the code itself is simple:
```  
h += golden_ratio_conjugate
h %= 1
out_colour = hsv_to_rgb(h, 0.5, 0.95)
```

I am wondering where to get hsv_to_rgb. The options are to use [a MIT-licensed code snippet from Stack overflow](https://stackoverflow.com/a/36209005), or to pull in OpenCV as a dependency.